CONTRIBUTING.md

```diff
@@ -306,9 +314,27 @@ $ ./cli exec --authn-oidc
 root@9feae5e5e001:/src/conjur-server#
 ```
 
+###### Run Cukes with Google Cloud Platform (GCP) Compatible Environment
+
+**Prerequisites**
+- A Google Cloud Platform account. https://cloud.google.com/.
+- Google Cloud SDK installed. https://cloud.google.com/sdk/docs
+- Access to a running Google Compute Engine instance. 
+
+To run the cukes with Google Cloud Platform (GCP) compatible environment, run `cli`
+with the `--authn-gcp` flag and pass a name of a running Google Compute Engine (GCE) instance:
+
+```sh-session
+$ ./cli exec --authn-gcp my-gce-instance
+...
+root@9feae5e5e001:/src/conjur-server#
+```
+
+When running with `--authn-gcp` flag, the cli script executes another script which does the heavy lifting of provisioning the ID tokens (required by the tests) from Google Cloud Platform.
+
 #### Run all the cukes:
 
-There are three different cucumber suites: `api`, `policy`, and `authenticators`. Each of these can be run using a profile of the same name:
+There are three different Cucumber suites: `api`, `policy`, and `authenticators`. Each of these can be run using a profile of the same name:
```

Comment:
and then how do I run gce specific tests? @eranha

Reply:
`cucumber -p authenticators_gce cucumber/authenticators_gce/features`

Reply:
don't we want to add them in the doc?